doc/next/_sources/reference/get-feed-import-id-detail.rst.txt

```diff
@@ -1,9 +1,84 @@
 .. index:: GET /feed/import/{id}/detail
+
 .. _get_feed_import_id_detail:
 
-GET /feed/import/{id}/detail
+GET /feed/import/{id}/detail V2
+============================
+
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``api_ro`` or ``console_ro``
+
+ * - Accept
+   - ``application/sellside.feedimportdetail.list-v2+json, application/json``
+
+This endpoint provides summary of the feed import.
+
+Each feed import object contains the time when it was started
+(``accepted``), the URL of the user's XML document (``url``) and whether the
+document could successfully be processed (``status``). The ``status`` field
+
+contains `0` when it is currently being processed status **PENDING**,
+
+contains `1` when the document was processed successfully status **DONE**,
+
+contains `2` when it was either not accessible or syntactically incorrect status **REJECTED**.
+
```

Comment:
why these spaces? Use table layout if you want to explain the 'status' field. Also, can't we put the enum's string value in there instead of numbers? String value here is much more readable.

Reply:
Sure, can do the table. What do you mean by `put the enum's string value ` ?